drivers/overlay/ov_network.go

```diff
@@ -46,10 +47,15 @@ type network struct {
 	once      *sync.Once
 	initEpoch int
 	initErr   error
-	subnets   []*subnet
+	*networkConfiguration
```

Comment:
I have taken care of the old data format stored in kv-store. See the test in overlay_test.go. 
The old format, eg `[{"SubnetIP":"10.0.0.0/24","GwIP":"10.0.0.1/24","Vni":256}]` is an array. So if we wants to add more options which are not per subnet, we have to change the format to a Map structure. That's what I do of adding a `networkConfiguration` structure.